locust/main.py

```diff
@@ -262,13 +262,124 @@ def parse_options():
         default=False,
         help="show program's version number and exit"
     )
+    return parser
+
+
+def create_options(
+        locustfile='locustfile',
```

Comment:
I wonder if there's a more reader friendly way to do this - or if the general use case will be ok with a massive list of kwargs here.

Reply:
Unsure, open to any suggestions.

Essentially, with the CLI tool, you're basically doing this same thing - loads of default arguments but the ability to change all of them via CLI arguments. But have put defaults in so unless the user who's calling this code only needs to put in the ones they're changing.